### PR TITLE
Add Airly documentation

### PR DESCRIPTION
**Pull request by bieniu**
**Description:**
Add dosumentation for Airly integration

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26375

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10285"></a>

### CODE REVIEW

File: source/_components/airly.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Airly"
+description: "Instructions on how to integrate Airly within Home Assistant."
+logo: airly.png
+ha_category:
+  - Health
+ha_release: 0.99
+ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/airly.markdown]: Please remove the `redirect_from`

File: source/_components/airly.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ - /components/sensor.airly/
+---
+
+The `Airly` integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location. To generate Airly API key go to [Airly for developers](https://developer.airly.eu/register) page.
+
+## Configuration
+
+To add Airly to your installation, go to Configuration >> Integrations in the UI and enable the Airly integration. By default the values will be taken from the Home Assistant configuration.
+
```
Comment (klaasnicolaas) [inline source/_components/airly.markdown:16]: Missing an example with required config variables

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/airly.markdown (hunk 2 above)
Comment (bieniu) [reply source/_components/airly.markdown:16]: This integration doesn't allow to configuration via `configuration.yaml` so there is no example.

Comment (klaasnicolaas) [reply source/_components/airly.markdown:16]: In that case we can skip my comment 😄

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in.

**Comment by bieniu** [comment]
@klaasnicolaas Could you review this PR once again?

File: source/_components/airly.markdown [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Airly"
+description: "Instructions on how to integrate Airly within Home Assistant."
+logo: airly.png
+ha_category:
+  - Health
+ha_release: 0.100
```
Comment (klaasnicolaas) [inline source/_components/airly.markdown]:
```suggestion
ha_release: "0.100"
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/airly.markdown [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Airly"
+description: "Instructions on how to integrate Airly within Home Assistant."
+logo: airly.png
+ha_category:
+  - Health
+ha_release: "0.100"
```
Comment (MartinHjelmare) [inline source/_components/airly.markdown]:
```suggestion
ha_release: 0.101
```

File: source/_components/airly.markdown [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+ha_iot_class: Cloud Polling
+---
+
+The `Airly` integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location. To generate Airly API key go to [Airly for developers](https://developer.airly.eu/register) page.
```
Comment (MartinHjelmare) [inline source/_components/airly.markdown]:
```suggestion
The `Airly` integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location. To generate an Airly API key go to [Airly for developers](https://developer.airly.eu/register) page.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @bieniu! 🎉

**Review by frenck — APPROVED** [review]

**Pull request merged**